vocabularies/Common.xml

```diff
@@ -113,7 +114,7 @@
         </Member>
       </EnumType>
 
-      <Term Name="Timezone" Type="Edm.String" AppliesTo="Property Parameter">
+      <Term Name="Timezone" Type="Edm.String" Nullable="false" AppliesTo="Property Parameter">
```

Comment:
```suggestion
      <Term Name="Timezone" Type="Edm.String" AppliesTo="Property Parameter">
```
Shall remain nullable, since it may be dynamic via `Edm.Path`.

Reply:
Then we should describe the desired behavior if the time zone is null: assume UTC?

Reply:
We have that already, see the LongDescription:
> In the absence of this annotation, points in time are typically interpreted in the current user's or default time zone.

Can be rephrased:
> If this annotation is absent or null, points in time ...

Reply:
Adapted text as proposed, set explicitly to nullable